keda/values.yaml

```diff
@@ -44,6 +44,8 @@ operator:
   # You can run multiple replicas, but they will not improve the performance of KEDA, it could only reduce downtime during a failover.
   # Learn more in [our documentation](https://keda.sh/docs/latest/operate/cluster/#high-availability).
   replicaCount: 1
+  # --Disable response compression for k8s restAPI in client-go. 
```

Comment:
That's a very technical thing to add; we may want to explain for non-Go experts/devs what it means

Reply:
sure @tomkerkhove

Reply:
any update on this?

Reply:
Hi @zroubalik , @tomkerkhove , I have added explanation for this.  PTAL!!